### DISCUSSION TITLE
Apply css to certain elements after choice?

### DISCUSSION THREAD

**Discussion by synapticlee** [jspsych/jsPsych#3036] [Q&A]
Hi, I have a 2AFC using html-keyboard-response in jspsych 6.3.1 and I am trying to apply css to the chosen stimulus after the participant chooses it (e.g. a fade, wiggle, grow, etc). I'm struggling with how to do this. The event-related callback function 'on_finish_trial' seems to be after the trial ends, when there are no elements left on the screen. How do I conditionally select the stimulus that was chosen and apply in-line css or something like that?

**Comment by ChristopheBossens** [comment]
Hi there,

In the html-keyboard-response, making a choice also ends the trial (unless you have the response_ends_trial variable set to true). So any css effects will not be visible, unless you start adjusting the plugin itself to make the stimulus stay on the screen longer after a response.

One approach you could try is to have two html-keyboard-response trials: one that requires a response and another one that only shows the same stimuli for a period of time. You can then use [dynamic parameters](https://www.jspsych.org/6.3/overview/dynamic-parameters/) for the second trial to apply a css style.
